uiowa/uiowa
# Now migration - link source based on redirects

Resolves https://github.com/uiowa/uiowa/issues/6021

# To Test
Normal migration setup. In devdesktop setup the now website locally from uiowa703
`ddev blt ds --site=now.uiowa.edu`
`ddev drush @now.local en now_migrate && ddev drush @now.local uli /admin/config/sitenow/sitenow-migrate`
fill in the migration data (now_uiowa_edu)
Add in the query code from https://github.com/uiowa/uiowa/pull/6141/commits/3009d0ee351e4c28ec7c7c13642cda42711782aa based on the findings from https://github.com/uiowa/uiowa/issues/6021#issuecomment-1409085398

`drush @now.local ms` should present you with 54 nodes to test against for the `now_news_feature` instead of testing the +6000k total nodes.

`drush @now.local mim now_news_feature`

As it runs, it should output the original node id and redirect to check now.uiowa.edu against. There should be 51 log messages of the 54 items migrated as three were skipped based on not being external URLs (based on the findings from https://github.com/uiowa/uiowa/issues/6021#issuecomment-1409085398).
